gradle/libs.versions.toml

```diff
@@ -0,0 +1,89 @@
+[versions]
+kotlin = "1.7.10"
+kotlin-coroutines = "1.6.4"
+java-target = "1.8"
+
+# Note: there are some compose/androidx versions left hardcoded in `paparazzi/paparazzi/build.gradle`,
+# because they refer to JAR files names in the `paparazzi/paparazzi/libs` folder.
+androidx-compose-compiler = "1.3.0"
```

Comment:
```suggestion
composeCompiler = "1.3.0"
```

Reply:
I went with `compose-compiler` to match style of other libs/versions, because there's a [ton of `compose` dependencies](https://mvnrepository.com/artifact/androidx.compose) and this would be the only one with camel-case in this project, everything else has dash separators. It fits in nicely if you look holistically.

Reply:
Versions camelcased.  Libraries dashes.

Reply:
ah